include/z64cutscene.h

```diff
@@ -150,7 +150,7 @@ typedef enum CutsceneCmd {
     /* 0x008F */ CS_CMD_ACTOR_CUE_9_0,
     /* 0x0090 */ CS_CMD_ACTOR_CUE_0_17,
     /* 0x03E8 */ CS_CMD_DESTINATION = 0x03E8,
-    /* 0xFFFF */ CS_CMD_END = 0xFFFF
+    /* 0xFFFF */ CS_CMD_END_OF_SCRIPT = -1 // 0xFFFF
```

Comment:
the hex comment should still be 0xFFFF

Reply:
I think it's a little weird adding `CS_CMD_NONE` since I don't think it's treated differently than any other "unknown" cutscene command in the code, but I don't feel strongly about it